sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/BlobFlatListSegment.java

```diff
@@ -21,14 +21,14 @@
      * The blobItems property.
      */
     @JsonProperty("Blob")
-    private List<BlobItem> blobItems = new ArrayList<>();
+    private List<BlobItemInternal> blobItems = new ArrayList<>();
```

Comment:
Need to consider if this is a breaking change and possibly revert

Reply:
It is not because we process this type internally and convert the elements into a PagedFlux. Even byPage on the paged flux doesn't use this.